dev-util/clion/clion-2018.2.6.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop eutils
+
+DESCRIPTION="A complete toolset for C and C++ development"
+HOMEPAGE="http://www.jetbrains.com/clion"
+SRC_URI="http://download.jetbrains.com/cpp/CLion-${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="IDEA
+	|| ( IDEA_Academic IDEA_Classroom IDEA_OpenSource IDEA_Personal )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+RESTRICT="splitdebug"
+IUSE="-custom-jdk"
```

Comment:
How is `-custom-jdk` different from `custom-jdk`? Legit question, not complaint.

Reply:
the minus in front of the use flag disables it per default

Reply:
But isn't it disabled by default? Then you can enable it with `+`. I don't think I've ever seen `-` being used there before :)

`[ebuild  N     ] dev-util/clion-2018.2.6::localhost  USE="-custom-jdk" 0 KiB`
with neutral IUSE. But once again I'm just wondering.

Reply:
That `-flag` thing is an obscure historical thingy that's supposed to workaround something. You really oughtn't use it in new ebuilds.